internal/output/html/filter_template.html

```diff
@@ -0,0 +1,85 @@
+<div id="summary-section">
+  <div id="filter-section">
+    {{ if .IsContainerScanning }}
+    <div class="filter-container">
+      <label for="layer-filter">Layer<br></label>
+      <div id="layer-filter" class="filter" onclick="toggleFilter('layer')">
+        <p value="all" id="layer-filter-selected" class="filter-selected">All layers ({{ .HTMLVulnCount.Called }})</p>
```

Comment:
issue: `value` is not allowed on most HTML tags, but you seem to be using it quite a lot - are you maybe meaning to use `title` instead?